Documentation/LSP Extensions.md

```diff
@@ -376,3 +376,47 @@ Users should not need to rely on this request. The index should always be update
 ```ts
 export interface TriggerReindexParams {}
 ```
+
+## `sourcekit-lsp/peekDocuments`
+
+Request from the server to the client to show the given documents in a "peeked" editor.
+
+This request is handled by the client to show the given documents in a "peeked" editor (i.e. inline with / inside the editor canvas).
+
+It requires the experimental client capability `"peekDocuments"` to use.
```

Comment:
I don’t think you currently have any capability check in place. Or am I missing something?

Reply:
https://github.com/swiftlang/sourcekit-lsp/pull/1479#discussion_r1660913419
@ahoppen @adam-fowler

Reply:
Ah, sorry. I forgot to remove this after I found the check via `initializeOptions`.